docs/API.md

```diff
@@ -25,63 +25,19 @@ Loads a document at the provided location and returns a JavaScript object repres
 
 **Example**  
 ```js
-// Example using Promises
-
-PathLoader
-  .load('./package.json')
-  .then(JSON.parse)
-  .then(function (document) {
-    console.log(document.name + ' (' + document.version + '): ' + document.description);
-  }, function (err) {
-    console.error(err.stack);
-  });
+// Example using Promises

PathLoader
  .load('./package.json')
  .then(JSON.parse)
  .then(function (document) {
    console.log(document.name + ' (' + document.version + '): ' + document.description);
  }, function (err) {
    console.error(err.stack);
  });
```

Comment:
Do you mind avoiding carriage returns?  I would assume that this could be done by using this: https://help.github.com/en/articles/dealing-with-line-endings#per-repository-settings  _(I'm not sure how this happened, I guess you're on Windows.)_

Reply:
Sure, let me see what I can do. I was a little confused when I saw those come in after the gulp command.

Reply:
And I am on Windows 😭